widget_driver/example/lib/widgets/coffee_community/coffee_library/coffee_detail/coffee_detail_page_driver.dart

```diff
@@ -9,8 +8,10 @@ part 'coffee_detail_page_driver.g.dart';
 class CoffeeDetailPageDriver extends WidgetDriver {
   final Coffee _coffee;
 
-  CoffeeDetailPageDriver(BuildContext context)
-      : _coffee = context.read<Coffee>(),
+  CoffeeDetailPageDriver(
+    BuildContext context, {
+    @driverProvidableModel required Coffee coffee,
+  })  : _coffee = coffee,
```

Comment:
```suggestion
  CoffeeDetailPageDriver(
    BuildContext context, {
    @driverProvidableModel  this.coffee,
  })  
```

Do we initialize as shown?

Reply:
Coffee is private in this class, this syntax only works on public fields.